.github/actions/setup-node/action.yaml

```diff
@@ -16,7 +16,7 @@ runs:
     - name: Setup Node
       uses: actions/setup-node@0a44ba7841725637a19e28fa30b79a866c81b0a6 # v4.0.4
       with:
-        node-version: 20.19.4
+        node-version: 22.14.0
```

Comment:
22.14 is actually older than 20.19, and would be a downgrade from tzdata 2025b to 2024b, which is why you're needing to filter out time zones that it's rejecting. is this the latest that nixpkgs offers?

ideally we'd upgrade to 22.19, which is the newest in the 22 lts line.